### PR TITLE
issue: 827622 Add jenkins support for inbox

### PR DESCRIPTION
**Pull request by igor-ivanov**
This change allows to check inbox/upstream
Also added few build configuration check such as
1. default
2. --enable-debug
3. --enable-opt-log

Signed-off-by: Igor Ivanov <igor.ivanov.va@gmail.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/241/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/256/ for details (Mellanox internal link).

File: contrib/jenkins_tests/globals.sh [hunk 1]
```diff
@@ -66,13 +66,31 @@ function do_github_status()
 
 function check_env()
 {
-	if [ -n "$ghprbTargetBranch" -a "$ghprbTargetBranch" != "master" ]; then
+    if [ $(command -v ofed_info >/dev/null 2>&1 || echo $?) ]; then
```
Comment (OphirMunk) [inline contrib/jenkins_tests/globals.sh]: Upstream driver may include ofed_info with "upstream" string included in OFED version name.
Please add this check to check_env(). Both upstream driver and inbox driver should be handled by Jenkins script

**Comment by OphirMunk** [comment]
For Jenkins to recognize Upstream/Inbox driver it is enough just to check existence of "ofed_info"
:+1:

**Comment by OphirMunk** [comment]
@igor-ivanov please let know when is ready to be merged (after concluding your tests on inbox setup)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/267/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/286/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/268/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/287/ for details (Mellanox internal link).

**Comment by igor-ivanov** [comment]
@OphirMunk it is ready

**Pull request merged**